components/atom/badge/src/index.scss

```diff
@@ -60,10 +62,14 @@ $badges: (
 
   &-icon {
     @include sui-icon--small;
-    display: inline-block;
+    display: inline-grid;
```

Comment:
Careful with that as it's not compatible with IE11. You must provide some kind of fallback or get a workaround for this.

Reply:
Fixed!